src/components/molecules/GenericMultiSelect.tsx

```
@@ -80,123 +98,136 @@ export const GenericMultiSelect = ({
+            )}
+          </div>
+        )}
+      </div>
+    );
+  };
+
   return (
     <div className="mb-4 text-white" ref={wrapperRef} onBlur={handleBlur}>
       {label && (
         <label className="block mb-1 text-sm font-medium">{label}</label>
       )}
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)